### ISSUE TITLE
Releasing in multiple languages

### ISSUE THREAD

**Issue by bvhme** [publiccodenet/standard#336]
There have been multiple people indicating the need for a release in their language.

This has the advantages:

* A lot easier to sell internally within public administrations
* Easier to get more people withing one public organization to join in on the development

And disadvantages:

* Every release needs to be translated into every supported language without exception
* Contributing back to the original version becomes a lot harder from a translation
* Reducing the ability to update and release new versions
* As the codebase needs to be in 'plain english' this introduces difficulties in multilingual cooperations

Languages that have been mentioned as requested:

* Dutch
* Italian
* German

Alternative stop-gap solutions could include:

* Executive summaries in multiple languages
* Non-official translations just to be used indicative

**Comment by bvhme** [comment]
As you can read, I'm critical if this is the right time for this. I've left out 'expensive' with the disadvantages since I think it might be misleading.

**Comment by 2045** [comment]
Hi There, I would very much like to help to translate into Dutch.. as we need instructions for our public servants.. If we cannot translate the one from public code, separate versions will pop-up .. We would rather link to the Dutch version of Public Code. I have done some translation for Mattermost, they used a tool named Poodle .. not sure if such a tool would help? Plz reach out how we can work on this..?

**Comment by bvhme** [comment]
That's great to hear @2045. As you know there are quite some difficulties with this and I personally am unclear whether the entire Standard for Public Code multiple languages is the right solution for now.

Perhaps we should have a community call about this specific issue.

If you have any drafts etcetera it would be great to see them in order to see what this might look like.

**Comment by marcvanandel** [comment]
Maybe in a growth scenario the foundation should have local communities? And they can support in translations and maintenance of the translations?

**Comment by 2045** [comment]
At the moment I am primarily focusing on 'the standard' PCN provides, as civil servants working with code need a standard / or maybe a better phrasing is 'guidance' (checklist) which helps them to determine how to open source their code, projects, etc.. in a safe and usable manner..
Reading and translating the PCN standard, I learn that it is more a kind of 'manifest' than a guideline how to open source in a proper and safe manner. (Example given, we have had engineer(s) putting certain keys on github.. which where identified as serious incidents.. so this engineer opened up, but slightly to much.. ) I am working on producing guidelines, helping PM's, PO's, Developers, etc. to open source their products in a safe and usable manner.

**Comment by Ainali** [comment]
Ping @2045 and @marcvanandel: In our next community call in one week (Thursday, August 6, 13.00 UTC) we will discuss this issue in more detail. [Agenda](https://hackmd.io/-OegeqvoThCbAsw3c3gIjw), [call in link](https://meet.publiccode.net/StandardforPublicCode). We will take notes and post here.

**Comment by 2045** [comment]
On holiday / vacation, I'll have the intention to join the meetings, after my holidays.. (as of 24 aug)

**Comment by Ainali** [comment]
Here are [the notes from the community call](https://blog.publiccode.net/community%20call/2020/08/24/notes-from-community-call-6-august-2020.html). 

TL;DR We will start looking in to how we can enable "community translations", ie. translations that are provided as a non-binding helpful service.

**Comment by Ainali** [comment]
As a result of the notes from the community call, here are some first assumptions / requirements for starting with translations:

1. It MUST be clear to the reader that it is the English language version that is used when checking for compliance and that all translations is a convenience service only.
2. We want to keep our main repository as untouched as possible and the contents of the text MUST remain unaffected. That means we will need to mirror the content (on every release?) to another repository to enable translations. This process will probably also include preparations that enables the translation.
3. To make translating easier, that repository should be connected to some external translation platform.
4. As much as possible should be automated, as in copied back and forth between the translation platform and the repository.
5. Preferably the community should be able to do as much as possible (after initial configuration). For instance, we want to empower the community to make releases in other languages when a translation is complete and not be blocked by us.
6. We want to make it easy and clear for the reader of a translated version to know which version in English it is translated from.

Are there any other important aspects we need before moving on to take a closer look at a process/platforms? Are my assumptions above correct?

**Comment by marcvanandel** [comment]
3. is optional; it's an implementation detail, right?

And maybe 'the community' should be 'a local community supporting the translation'?

**Comment by Ainali** [comment]
> 1. is optional; it's an implementation detail, right?
> 
> And maybe 'the community' should be 'a local community supporting the translation'?

Regarding 3. You are right, it is optional (hence the should). This is a perfect thing to discuss in the community call tomorrow.

I think I mean the wider community. I see a possibility for people being interested in supporting the releases for translations without being involved in the actual translation. The work involved in making that requires another skill set (and possibly interest) than just doing translations. Obviously, there will be a need for communication to know when something is ready for release.

**Comment by marcvanandel** [comment]
> 
> Regarding 3. You are right, it is optional (hence the should). This is a perfect thing to discuss in the community call tomorrow.
> 

The SHOULD wasn't capitals (like the MUST) so I overlooked it a little bit 😁

**Comment by nicorikken** [comment]
At the FSFE we have plenty of texts, including news articles, that are translated by many volunteers. While much of the review process occurs on the `translators@` mailinglist, the source is handled on git: https://git.fsfe.org/FSFE/fsfe-website/src/branch/master/news/2020 Recently we started versioning texts and adding a version to the translation as well. This allows us to track translations that have become out of date: https://status.fsfe.org/translations.html

From other communities I've heard positive remarks on translation platforms regarding usability. The FSFE sets a high bar for tools they adopt, so I think it is likely we'll keep this DIY solution for the near future.

**Comment by Ainali** [comment]
We forgot to post the outcome from our community call on 3 September, [but here they are](https://blog.publiccode.net/community%20call/2020/09/22/notes-from-community-call-3-september-2020.html).

**Comment by cheikendorf** [comment]
Following the idea af translations as convenience service only, I have translated the Standard (pretty much all of it) into Danish. I am not at professional translater, and this therefore falls in the group of 'local community translation'.
[1 Introduktion og baggrund.docx](https://github.com/publiccodenet/standard/files/5438131/1.Introduktion.og.baggrund.docx)

**Comment by ericherman** [comment]
Thank you @cheikendorf for posting your Danish draft!

**Comment by cheikendorf** [comment]
> 
> 
> Thank you @cheikendorf for posting your Danish draft!

No problem. This is only the very first section in the Standard.

**Comment by Ainali** [comment]
We now have the [community-translations-standard](https://github.com/publiccodenet/community-translations-standard) repository for this!
